src/components/DetailSnippet/SnippetTool/SnippetTool.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+import Button from "../../common/Button";
+
+import VARIANTS from "../../../constants/variants";
+
+const { TOOL } = VARIANTS;
+
+const ToolWrapper = styled.div`
+`;
+
+export default function SnippetTool() {
+  return (
+    <ToolWrapper>
+      <Button variant={TOOL}>저장</Button>
+      <Button variant={TOOL}>복사</Button>
+      <Button variant={TOOL}>공유</Button>
+      <Button variant={TOOL}>삭제</Button>
```

Comment:
map 메서드를 이용하면 더 간단하게 작성할 수 있습니다.

```suggestion
      {["저장", "복사", "공유", "삭제"].map((name) => (
        <Button variant={TOOL}>{name}</Button>
      ))}
```